applications/luci-app-cloudflared/root/usr/share/rpcd/acl.d/luci-app-cloudflared.json

```diff
@@ -6,11 +6,17 @@
 			"ubus": {
 				"service": [ "list" ]
 			},
-			"file": [ "/var/log/cloudflared.log" ]
+			"file": {
+				"/etc/cloudflared/*": [ "read" ],
+				"/var/log/cloudflared.log": [ "read" ]
+			}
 		},
 		"write": {
 			"uci": [ "cloudflared" ],
-			"file": [ "/etc/cloudflared/" ]
+			"file": {
```

Comment:
On the principle of giving the least necessary permissions, please don't do this. Use the permissions I outlined earlier in #6834 
```json
		"write": {
			"uci": ["cloudflared"],
			"file": {
				"/etc/cloudflared/*": ["read"],
				"/etc/cloudflared/config.yml": ["write"],
				"/etc/cloudflared/config.yaml": ["write"],
				"/etc/cloudflared/cert.pem": ["write"],
			}
```

@animegasan can you verify that the above permissions context also works for your test case?

Reply:
> On the principle of giving the least necessary permissions, please don't do this. Use the permissions I outlined earlier in #6834
> 
> ```json
> 		"write": {
> 			"uci": ["cloudflared"],
> 			"file": {
> 				"/etc/cloudflared/*": ["read"],
> 				"/etc/cloudflared/config.yml": ["write"],
> 				"/etc/cloudflared/config.yaml": ["write"],
> 				"/etc/cloudflared/cert.pem": ["write"],
> 			}
> ```
> 
> @animegasan can you verify that the above permissions context also works for your test case?

I used it like this and it worked
```json
                "write": {
                        "uci": [ "cloudflared" ],
                        "file": {
                                "/etc/cloudflared/config.yml": ["write"],
                                "/etc/cloudflared/config.yaml": ["write"],
                                "/etc/cloudflared/cert.pem": ["write"],
                                "/usr/bin/cloudflared *": [ "exec" ]
                        }
                }
```

Reply:
I don't see a problem that the entire /etc/cloudlfared is accessible for the app. I may have a `config.yml.bak` file there and specify it. Other parts of system won't be affected.
So let's leave it as is.